api/v1alpha1/rediscacheaction_types.go

```diff
@@ -51,3 +56,15 @@ type RedisCacheActionList struct {
 func init() {
 	SchemeBuilder.Register(&RedisCacheAction{}, &RedisCacheActionList{})
 }
+
+func IsRedisCacheRollAction(actionName RedisCacheActionName) bool {
+	return actionName == RedisCacheActionNameRollAllKeys ||
+		actionName == RedisCacheActionNameRollPrimaryKey ||
+		actionName == RedisCacheActionNameRollSecondaryKey
+}
+
+func IsRedisCacheRebootAction(actionName RedisCacheActionName) bool {
+	return actionName == RedisCacheActionNameRebootAllNodes ||
+		actionName == RedisCacheActionNameRebootPrimaryNode ||
+		actionName == RedisCacheActionNameRebootSecondaryNode
+}
```

Comment:
I wouldn't block on this, but I'm not in love with putting these functions here... it seems to me they should be in the reconcile or manager files - again, that's probably preference though

Reply:
I put them there because that is where the enum/string alias is defined so I thought it would be easier to change if you update the enum but I can move them to the reconciler.